src/main/resources/default/assets/wondergem/autocomplete/multiselect.js

```diff
@@ -215,6 +217,34 @@ var multiSelect = function (args) {
                 tokenfield.getTokenfieldInputField()[0].placeholder = args.searchKey || args.placeholder;
             });
 
+            $('#' + args.id).on('keyup', '.tokenfield', function (e) {
+                if (e.target !== e.currentTarget) {
+                    // only responde to key-events off the container
+                    return;
+                }
+                if (e.keyCode !== sirius.keys.KEY_ENTER
```

Comment:
why not flip if condition? less checks on average and easier to read because less negations